shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/core/NetworkClient.kt

```diff
@@ -53,7 +57,14 @@ open class NetworkClient {
 
     private fun clientConfig(): HttpClientConfig<*>.() -> Unit {
         return {
-            install(Logging)
+            install(Logging) {
+                level = LogLevel.ALL
+                logger = object : Logger {
+                    override fun log(message: String) {
+                        Napier.log(io.github.aakira.napier.LogLevel.DEBUG, message = message)
```

Comment:
We can import `import io.github.aakira.napier.LogLevel` then
```suggestion
                        Napier.log(LogLevel.DEBUG, message = message)
```